packages/app-page-builder/src/editor/components/Text/PbText.tsx

```diff
@@ -59,7 +59,10 @@ const PbText: React.FC<TextElementProps> = ({ elementId, mediumEditorOptions, ro
         return null;
     }
 
-    const textContent = variableValue || get(element, `${DATA_NAMESPACE}.data.text`);
+    // only used to set initial value, so no reason to update it every time
+    const textContent = useMemo(() => {
```

Comment:
BTW, can we rename this variable then to `initialText`? That way even the comment would not be needed.

Reply:
Also, this hook is used after the if statement that can potentially return `null` (check L59).

Because of this, this hook usage should be relocated above the if statement.

Reply:
@adrians5j I have changed it according to a comment.
Also I have dropped `return` statement (to be in same style as `fallbackValue`)